packages/perseus-editor/src/components/circle-swatch.tsx

```diff
@@ -0,0 +1,63 @@
+import {lockedFigureColors, lockedCircleFillStyles} from "@khanacademy/perseus";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {color as wbColor, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import type {
+    LockedFigureColor,
+    LockedCircleFillType,
+} from "@khanacademy/perseus";
+
+type Props = {
+    color: LockedFigureColor;
+    fillStyle: LockedCircleFillType;
+    strokeStyle: "solid" | "dashed";
+};
+
+const CircleSwatch = (props: Props) => {
+    const {color, fillStyle, strokeStyle} = props;
+
+    return (
+        <View
+            aria-label={`${color}, stroke ${strokeStyle}, fill ${fillStyle}`}
```

Comment:
I added it so that the accordion header label would just include aspect of the line summary. I know we don't have any content author using screen readers as far as I know, so I mostly added this so I can use `getByLabelText` in jest tests.